recon/src/recon/tests.rs

```diff
@@ -416,378 +418,379 @@ fn hello() {
-                            1341358262,
-                            1792645756,
-                            2297177231,
-                            2397729726,
-                            644181082,
-                            2813893646,
-                        ],
-                    },
+    Recon {
+        store: BTreeStore {
+            keys: {
+                Bytes(
+                    "hello",
+                ): Sha256a {
+                    hex: "2CF24DBA5FB0A30E26E83B2AC5B9E29E1B161E5C1FA7425E73043362938B9824",
+                    u32_8: [
+                        3125670444,
+                        245608543,
+                        708569126,
+                        2665658821,
+                        1545475611,
+                        1581426463,
+                        1647510643,
+                        613976979,
+                    ],
+                },
+                Bytes(
+                    "world",
+                ): Sha256a {
+                    hex: "486EA46224D1BB4FB680F34F7C9AD96A8F24EC88BE73EA8E5A6C65260E9CB8A7",
+                    u32_8: [
+                        1654943304,
+                        1337708836,
+                        1341358262,
+                        1792645756,
+                        2297177231,
+                        2397729726,
+                        644181082,
+                        2813893646,
+                    ],
                 },
             },
-        }
-    "#]]
+        },
+    }
+"#]]
     .assert_debug_eq(&other_hash)
 }
 
 #[test]
 fn abcde() {
     recon_test(expect![[r#"
-        cat: [b,c,d,e]
-        dog: [a,e]
-        -> (b, h(c,d), e) [a,b,e]
-        <- (a, 0, b, 0, e) [a,b,c,d,e]
-        -> (a, 0, b, 0, c, 0, d, 0, e) [a,b,c,d,e]
-        <- (a, h(b,c,d), e) [a,b,c,d,e]"#]])
+    cat: [b,c,d,e]
+    dog: [a,e]
+    -> (b, h(c,d), e) [a,b,e]
+    <- (a, 0, b, 0, e) [a,b,c,d,e]
+    -> (a, 0, b, 0, c, 0, d, 0, e) [a,b,c,d,e]
+    <- (a, h(b,c,d), e) [a,b,c,d,e]"#]])
 }
 
 #[test]
 fn two_in_a_row() {
     recon_test(expect![[r#"
-    cat: [a,b,c,d,e]
-    dog: [a,d,e]
-    -> (a, h(b,c,d), e) [a,d,e]
-    <- (a, 0, d, 0, e) [a,b,c,d,e]
-    -> (a, 0, b, 0, c, h(d), e) [a,b,c,d,e]
-    <- (a, h(b,c,d), e) [a,b,c,d,e]"#]])
+cat: [a,b,c,d,e]
+dog: [a,d,e]
+-> (a, h(b,c,d), e) [a,d,e]
+<- (a, 0, d, 0, e) [a,b,c,d,e]
+-> (a, 0, b, 0, c, h(d), e) [a,b,c,d,e]
+<- (a, h(b,c,d), e) [a,b,c,d,e]"#]])
 }
 
 #[test]
 fn test_parse_recon() {
     let recon = r#"
-        cat: [a,b,c]
-        dog: [e,f,g]
-        -> (a, h(b), c) [a,c,e,f,g]
-        <- (a, 0, c, h(e,f), g) [a,b,c,g]
-        -> (a, 0, b, 0, c, 0, g) [a,b,c,e,f,g]
-        <- (a, h(b), c, 0, e, 0, f, 0, g) [a,b,c,e,f,g]
-        "#;
+    cat: [a,b,c]
+    dog: [e,f,g]
+    -> (a, h(b), c) [a,c,e,f,g]
+    <- (a, 0, c, h(e,f), g) [a,b,c,g]
+    -> (a, 0, b, 0, c, 0, g) [a,b,c,e,f,g]
+    <- (a, h(b), c, 0, e, 0, f, 0, g) [a,b,c,e,f,g]
+    "#;
```

Comment:
why have all these test cases changed their whitespace?

Reply:
This was the result of `cargo fmt --all`. Maybe originally formatted with an older fmt?

Reply:
CI runs `cargo fmt --all -- --check` to ensure everything is formatted and runs on latest Rust release. Wonder why it changed.

Reply:
I didn't know cargo fmt messed with`r#""#` string literals.

Reply:
Figured it out, some of it was fmt, some of it was IDE auto formatting.